zookeeper-assembly/src/main/assembly/lib-package.xml

```diff
@@ -54,7 +54,7 @@
       <includes>
         <include>LICENSE</include>
       </includes>
-      <outputDirectory>/</outputDirectory>
```

Comment:
how did this work before? and why is . better?

Reply:
The maven-assembly-plugin tries to interpret the value of this field relative to the root of the archive being assembled. On Linux, a `/` would result in the files being placed at the root of the archive. However, according to the warning from the maven-assembly-plugin, this relative-path resolution using an absolute path value (one starting with a slash) is not platform-dependent. Using `.` here achieves the intended goal of placing the files at the root of the archive, but does not generate the warning from the plugin.